website/source/guides/upgrade/upgrade-specific.html.md

```diff
@@ -15,6 +15,18 @@ details provided for their upgrades as a result of new features or changed
 behavior. This page is used to document those details separately from the
 standard upgrade flow.
 
+## Nomad 0.10.0
+
+### Deployments
+
+Nomad 0.10 enables rolling upgrade deployments for jobs by default,
+adding a default update stanza when you create and update jobs
+that do not already have an update stanza present.
+This does not affect existing jobs with an update stanza.
+In pre-0.10.0 releases, jobs without an update stanza were deployed
+with destructive updates by default. To regain this behavior,
+see [`update stanza`][update-stanza].
```

Comment:
This didn't render correctly

Reply:
@jazzyfresh you will need to add a link at the end for update stanza. See the end of this file for how the other URLS are defined.